### PR TITLE
Update TROUBLESHOOTING.md to include language server stopping guide

### PR DESCRIPTION
**Pull request by mgmanzella**
Encountered an issue with theme-check locally and updated docs with some hints on how to debug 😄

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -46,5 +46,20 @@ chruby 3.1.2 #your theme ruby version
 bundle install
 ```
 
+### Language server sends an `initialize()` request to the client and stops
+
+To debug, confirm these steps are included in your local language server startup script:
+
+```json
```
[the remark this answers is not in the document]
Comment (mgmanzella) [reply TROUBLESHOOTING.md:53]: 🤦 ty!

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -46,5 +46,20 @@ chruby 3.1.2 #your theme ruby version
...
+
+To debug, confirm these steps are included in your local language server startup script:
+
+```json 
+export THEME_CHECK_DEBUG=true
+export THEME_CHECK_DEBUG_LOG_FILE="/tmp/theme-check-debug.log"
+touch "$THEME_CHECK_DEBUG_LOG_FILE"
+```
+
+An example script can be found [here](/CONTRIBUTING#run-language-server).
```
[the remark this answers is not in the document]
Comment (mgmanzella) [reply TROUBLESHOOTING.md:59]: hmmmm weird so that syntax _should_  link to Run Language Server in CONTRIBUTING but let me try again

**Review by Poitrin — APPROVED** [review]
LGTM :)
